app/test_main.py

```diff
@@ -1,3 +1,57 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+class TestAnimalToHumanAge:
+    def test_result_list_length_should_be_equal_to_2(self) -> None:
+        assert len(get_human_age(1, 1)) == 2
+
+    @pytest.mark.parametrize(
+        "cat_age,dog_age,expected_human_ages",
+        [
+            pytest.param(0, 0, [0, 0], id="should be zeros if ages less than 15"),
+            pytest.param(14, 14, [0, 0], id="should be zeros if ages less than 15"),
+            pytest.param(15, 15, [1, 1], id="should be [1, 1] if ages between 15 and 23"),
+            pytest.param(23, 23, [1, 1], id="should be [1, 1] if ages between 15 and 23"),
+        ]
+    )
+    def test_cat_and_dog_ages(self, cat_age, dog_age, expected_human_ages):
+        result = get_human_age(cat_age, dog_age)
+        assert expected_human_ages == result
+
+    @pytest.mark.parametrize(
+        "cat_age,cat_to_human_age",
+        [
+            pytest.param(24, 2, id="every 4 cats years should be converted to 1 if age greater than 23"),
+            pytest.param(27, 2, id="every 4 cats years should be converted to 1 if age greater than 23"),
```

Comment:
These two test cases are testing the same scenario: when the cat age is greater than 23, every 4 years should convert to 1 human year. Consider removing one of them to avoid redundancy.